src/plugins/spark/features/Pid/PidWidget.vue

```diff
@@ -86,15 +123,15 @@ export default class PidWidget extends BlockWidget {
         <q-separator dark inset class="q-mb-md"/>
       </template>
 
-      <q-item dark>
+      <q-item dark clickable @click="inputFormOpen = true">
```

Comment:
Bit of a mismatch: the field is made to look clickable (underlined), but the whole q-item is clickable.